src/instrument.js

```diff
@@ -70,7 +70,7 @@ function computeNextEntry(reducer, action, state, error) {
   let nextState = state;
   let nextError;
   try {
-    nextState = reducer(state, action);
+    nextState = reducer(state, { ...action, replaying });
```

Comment:
What if the user already has a meaning for `replaying` field? Can this be a Symbol instead?